apps/webapp/app/hooks/useSearchParam.ts

```diff
@@ -18,13 +18,13 @@ export function useSearchParams() {
         }
 
         if (typeof value === "string") {
-          search.set(param, encodeURIComponent(value));
+          search.set(param, value);
           continue;
         }
 
         search.delete(param);
         for (const v of value) {
-          search.append(param, encodeURIComponent(v));
+          search.append(param, v);
```

Comment:
This fixes an unrelated issue with the run filters where they were being double uri encoded. This meant filters with weird characters in (like tasks with `/` weren't working with the filters).

Reply:
This isn't needed. URLSearchParams automatically does this when using `set` and then doing .toString() on the searchParams. I was doing it before and it was double encoding them which caused problems.